submissions/shramkoweb/js-dom/main.css

```diff
@@ -0,0 +1,73 @@
+html {
```

Comment:
```suggestion
*,
*::before,
*::after {
     box-sizing: border-box;
}
```

Reply:
I think my is better. Because inheritance is more flexible in feature.
If I wrong can u explain why you variant better ?